build/style.css

```diff
@@ -100,3 +104,32 @@ strong {
 	opacity: 0.6;
 	border-bottom-color: #bb5536;
 }
+
+div.main.wrapper h2 {
+	color: #b30000;
+}
+
+div.wrapper h2.trouble {
+	float: right;
+	padding: 20px 0px;
+	color: #a9a9a9;
+	text-transform: uppercase;
+}
+div.article.full.section.content{
+	display: flex;
+}
+
+div.side-bar{
```

Comment:
rename to `.side-bar` and then remove both `align-items` and `width` and just do the `flex: 1 0 0;` that we talked about.